src/main/resources/org/codehaus/mojo/servicedocgen/generation/velocity/OpenApi.json.vm

```diff
@@ -0,0 +1,154 @@
+{
+  'openapi': '3.0.0',
+  'info': {
+    'title': '$services.info.title',
+    'version': '$services.info.version',
+    'description': '$services.info.description',
+    'contact': {
+      'name': '$services.info.contact.name',
+      'url': '$services.info.contact.url',
+      'email': '$services.info.contact.email',
+    },
+    'license': {
+      'name': '$services.info.license.name',
+      'url': '$services.info.license.url'
+    }
+  },
+  'paths': {
+#foreach ($service in $services.services)
+#foreach ($operation in $service.operations)
+#set( $nextIndex = $foreach.index + 1)
+#set( $previousIndex = $foreach.index - 1)
+#set( $listLength = $service.operations.size())
+#if($foreach.index == 0 || $service.operations.get($foreach.index).path != $service.operations.get($previousIndex).path)
+    '$service.basePath$operation.path': {
+#end
+      '$operation.httpMethod': {
+        'description': '$operation.description',
+        'operationId': '$operation.id',
+#if($operation.httpMethod == "post" && $operation.parameters.size() == 1)
+        'requestBody': {
+          'content': {
+            '$display.list($operation.produces)': {
+              'schema': {
+#if($operation.getParameters().get(0).javaParameter.getSourceType().isPrimitive())
```

Comment:
when calling a simple getter you should avoid using method style for simplicity and readability.
So instead of `.getSourceType()` just write `.sourceType`. and instead of `isPrimitive()` write `primitive`.

This also applies for all further template code...